openbb_terminal/portfolio/portfolio_controller.py

```diff
@@ -638,6 +643,112 @@ def call_alloc(self, other_args: List[str]):
                         f"are: {', '.join(self.AGGREGATION_METRICS)}"
                     )
 
+    @log_start_end(log=logger)
+    def call_attrib(self, other_args: List[str]):
+        """Process attrib command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="attrib",
+            description="""
+                Displays sector attribution of the portfolio compared to the S&P 500            
+                """,
+        )
+        parser.add_argument(
+            "-p",
+            "--period",
+            type=str,
+            choices=portfolio_helper.PERIODS,
+            dest="period",
+            default="all",
+            help="The file to be loaded",
+        )
+        parser.add_argument(
+            "-t",
+            "--type",
+            type=str,
+            choices=["relative","absolute"],
+            dest="type",
+            default="relative",
+            help="select between relative or absolute attribution values",
+        )
+        parser.add_argument(
+            "-r",
+            "--raw",
+            type=bool,
+            dest="raw",
+            default=False,
+            const=True,
+            nargs="?",
+            help="view attribution in a graph visualisation format",
```

Comment:
With `-r` you show the table + graph, this does not disable the graph.

Reply:
For this we were using -r as an abbreviation for "--raw". Regarding the showing of the graph, based on what was mentioned earlier we were under the impression that "--raw" is only to show or disable the chart not the graph.

Please let us know if we have misunderstood the request and what behavior you would like the "-r"/"--raw" flag to exhibit.

![image](https://user-images.githubusercontent.com/80865650/201008596-59a7e3ff-9d48-4f78-ae9f-08250fd986f0.png)

Reply:
So, when you call `--raw`, I want to see the `table` and this is exactly what happens. However, the `help` argument says: `help="view attribution in a graph visualisation format"` and that's incorrect, you are actually activating the table by calling `--raw` not the graph so this is mostly a documentation thing.